binds/python/bind_mutable.cpp

```diff
@@ -115,7 +115,9 @@ void bind_mutable_module(py::module& m) {
         .def_property_readonly("version", &morphio::mut::Morphology::version, "Returns the version")
 
         .def("write",
-             &morphio::mut::Morphology::write,
+             [](morphio::mut::Morphology* morph, py::object arg) {
```

Comment:
I use pointers only when it is necessary to manipulate the object's address rather than the object itself. But this could be done in a separate PR to apply this to all bindings.

So more something like this:
```c++
             [](morphio::mut::Morphology& morph, const py::object& arg) {
```